src/cli/domain/package-template.js

```diff
@@ -20,21 +20,22 @@ var javaScriptizeTemplate = function(functionName, data){
   return format('var {0}={0}||{};{0}.components={0}.components||{};{0}.components[\'{1}\']={2}', 'oc', functionName, data.toString());
 };
 
-var compileView = function(viewPath, type) {
+var compileView = function(viewPath, type, cb) {
   var template = fs.readFileSync(viewPath).toString();
 
   if(!templateEngines[type]){
     throw strings.errors.cli.TEMPLATE_TYPE_NOT_VALID;
   }
 
-  var compiledView = templateEngines[type].compile(template, { viewPath }),
-      hashView = hashBuilder.fromString(compiledView.toString()),
-      javaScriptizedView = javaScriptizeTemplate(hashView, compiledView);
+  var compiledView = templateEngines[type].compile({ template, viewPath }, compiledView => {
```

Comment:
=> are still currently illegal according to jshint styleguide. While we upgrade to ES6, can you switch this statement to ES5?